src/clojisr/v1/require.clj

```diff
@@ -69,45 +76,64 @@
         (seq opt) (list ['& {:keys opt}])
         :else '([])))))
 
-(defn r-symbol->clj-symbol [r-symbol r-object]
+(defn- safe-help [r-object]
+  (try 
+    (help/help r-object)
+    (catch Exception e "")))
+
+(defn r-symbol->clj-symbol [ r-symbol r-object] 
+  
   (if-let [arglists (r-object->arglists r-object)]
     (vary-meta r-symbol assoc :arglists arglists)
     r-symbol))
 
+
+(defn- assoc-doc-to-meta! [ns-symbol r-symbol r-object]
+  (alter-meta!
+   (get (ns-publics ns-symbol) r-symbol)
+   assoc :doc (safe-help r-object)))
+
 (defn add-to-ns [ns-symbol r-symbol r-object]
   (intern ns-symbol
           (r-symbol->clj-symbol r-symbol r-object)
-          r-object))
+          r-object)
+  ns-symbol)
+
 
 (defn symbols->add-to-ns [ns-symbol r-symbols]
   (doseq [[r-symbol r-object] r-symbols]
-    (add-to-ns ns-symbol r-symbol r-object)))
+    (add-to-ns ns-symbol r-symbol r-object))
+  (future
+    (Thread/sleep 5000) ;; this has the effect that the evaluatin of r-require "returns immidiately" during interactive work
```

Comment:
`future` runs just after symbols are interned - do we need this 5sec waiting time?

Reply:
Without it we have a noticablr lag in IDE when interactively evaluating a 'require-r'